index.js

```diff
@@ -81,18 +100,36 @@ const getButton = document.getElementById(
   'submit'
 );
 
+// rewrite Promise with async/await
+
 getButton.addEventListener(
   'click',
-  () => {
-    flushAvatars();
-    fetchUsers(readInput()).then(
-      users => {
-        users.forEach(user => {
-          displayAvatar(
-            getAvatar(user)
-          );
-        });
-      }
-    );
+  async () => {
+    try {
+      flushAvatars();
+      let users = await fetchUsers(readInput());
+      users = users.forEach( user => {
```

Comment:
I think we don't need to reassign `users` here cause `forEach` doesn't return anything.
```suggestion
      users.forEach( user => {
        displayAvatar(getAvatar(user));
        });
```